### PR TITLE
Create OperationsCatalog.md

### PR DESCRIPTION
**Pull request by papollosc**
Added feedback form

### CODE REVIEW

File: preview-docs/OperationsCatalog.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+2. **Share with other members of the team:** Developers can share the collection/s of operations with other members of the team that have at least read access to the graph. For public graphs, sharing can be possible outside of the members of the Studio Org. Sharing of collections is made possible by:
+    1. Share the collection directly with other members of the org that have access to Studio: The shared collection will be visible in the operations catalog of each member that it is shared with. Any user with the permission to edit the collection can make changes to the collection or any of the operation. Sharing can also be done as view-only, where the user with view-only permission cannot make any changes to the collection.
+    2. Share the collection with a link: A unique link for that collection will be generated that can be distributed for sharing. Anyone who has access to the link will be able to view the collections. Link sharing can be used for sharing a collection outside of the Studio org. Link sharing only gives view-only access to the collections. The link is always up to date with the changes made to the collection.
```
Comment (papollosc) [inline preview-docs/OperationsCatalog.md:20]: "For public graphs" link sharing can be used for sharing a collection outside of the Studio org.

Comment (papollosc) [reply preview-docs/OperationsCatalog.md:20]: The link will open up in Apollo Sandbox?

File: preview-docs/OperationsCatalog.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+3. **Reference and download:** A GraphQL document that represents the collection can also be easily exported for distribution. A particular operation or collection of operations can also be used in markdown for inline rendering. 
+4. **Production data access:** Certain operations or collections can be marked as “secure”, meaning that they are good for pulling production data, allowing non-technical users who want to query graph data easy access to parameterize those operations. A specific list of users or a user with a secret token can query data using secure operations/collections.
+5. **Schema checks:** A collection of operations can be used as part of an integration test suite or to ensure that a given schema state is OK to deploy by automatically validating whether the given schema state can fulfill that collection of operations.
```
Comment (papollosc) [inline preview-docs/OperationsCatalog.md:24]: An operation or a collection of operations

File: preview-docs/OperationsCatalog.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+6. **Secure gateway:** A collection of operations can be used to block the gateway so that only certain types and shapes of operations can make it through.
```
Comment (papollosc) [inline preview-docs/OperationsCatalog.md:25]: An operation or a collection of operations

File: preview-docs/OperationsCatalog.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+1. **Save and organize operations as collections:** A collection is a set of operations. Developers will be able to save the operations that they write in Studio for later use and easy access and organize them as collections. They can create, update, delete and read the collections of operations. Collections will have documentation that can be used to explain their purpose. Comments on collections for collaboration will also be made available. 
+2. **Share with other members of the team:** Developers can share the collection/s of operations with other members of the team that have at least read access to the graph. For public graphs, sharing can be possible outside of the members of the Studio Org. Sharing of collections is made possible by:
+    1. Share the collection directly with other members of the org that have access to Studio: The shared collection will be visible in the operations catalog of each member that it is shared with. Any user with the permission to edit the collection can make changes to the collection or any of the operation. Sharing can also be done as view-only, where the user with view-only permission cannot make any changes to the collection.
```
Comment (daniman) [inline preview-docs/OperationsCatalog.md:19]: Do you provide permission on the collection on a user basis? That may be a rabbit hole with respect to database modeling. Could we start with the simpler approach of "graph admins+ can edit all collections" or something?

File: preview-docs/OperationsCatalog.md (hunk 1 above)
Comment (daniman) [reply preview-docs/OperationsCatalog.md:20]: Do the collections have any relationship to the specific graph? Are they modeled as a dependent of a graph, or are they completely independent entities?

File: preview-docs/OperationsCatalog.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+    2. Share the collection with a link: A unique link for that collection will be generated that can be distributed for sharing. Anyone who has access to the link will be able to view the collections. Link sharing can be used for sharing a collection outside of the Studio org. Link sharing only gives view-only access to the collections. The link is always up to date with the changes made to the collection.
+    3. Share the collection with a run in Explorer button: Using an explorer button, operations collections can be embedded for use in documentation. Using this button, a developer can either import the collection and use it in their own IDE or they can open the collection in Apollo Sandbox to execute. The run button for collection is always up to date with the changes made to the collection.
```
Comment (daniman) [inline preview-docs/OperationsCatalog.md:21]: Oh, love the concept of this.

File: preview-docs/OperationsCatalog.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+3. **Reference and download:** A GraphQL document that represents the collection can also be easily exported for distribution. A particular operation or collection of operations can also be used in markdown for inline rendering. 
+4. **Production data access:** Certain operations or collections can be marked as “secure”, meaning that they are good for pulling production data, allowing non-technical users who want to query graph data easy access to parameterize those operations. A specific list of users or a user with a secret token can query data using secure operations/collections.
```
Comment (daniman) [inline preview-docs/OperationsCatalog.md:23]: Would a collection potentially span graphsn and variants? So you could save a collection that included some operations to prod, some operations to staging, and some operations to another graph entirely?

File: preview-docs/OperationsCatalog.md (hunk 3 above)
Comment (daniman) [reply preview-docs/OperationsCatalog.md:25]: Are you thinking this is potentially a new way to do Operation Safelisting?

**Review by daniman — COMMENTED** [review]
Will collections work in sandbox, or just for deployed graphs? will they work for dev graphs?

File: preview-docs/OperationsCatalog.md [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+
+As part of this developer toolkit, we provide Apollo Studio Explorer, where developers can create, run and manage operations executed against the graph. _A critical piece of this experience is the ability to save, organize and share operations, which enables cross team collaboration._
+
+Following are some of the use cases that we are considering for solutioning operations catalog in Apollo Studio. We would love to get your feedback on whether these will be useful for your organization and how they might fit in graph exploration, collaboration and consumption experience. If you're reading this, please fill out our feedback form! We are grateful for your feedback and read every response that comes in: (insert feedback form link).
```
Comment (papollosc) [inline preview-docs/OperationsCatalog.md:13]: https://www.surveymonkey.com/r/ApolloOperationsCatalog

File: preview-docs/OperationsCatalog.md (hunk 3 above)
Comment (papollosc) [reply preview-docs/OperationsCatalog.md:25]: Yes, that is what I got from the draft of operations catalog that Nathan had https://apollographql.quip.com/YfjJA18wat3K/Product-Brief-Operation-Catalog

File: preview-docs/OperationsCatalog.md (hunk 4 above)
Comment (papollosc) [reply preview-docs/OperationsCatalog.md:19]: ohh good point. Yes love the modification.

File: preview-docs/OperationsCatalog.md (hunk 3 above)
Comment (daniman) [reply preview-docs/OperationsCatalog.md]: beautiful, love it

**Comment by papollosc** [comment]
@daniman Please go ahead and merge when suitable!

**Pull request merged**